.github/workflows/rust_build_ockam_ockam_channel.yml

```diff
@@ -0,0 +1,52 @@
+name: rust_build_ockam_ockam_channel
```

Comment:
This needs to be updated, since then we greatly simplified our gha workflows and you shouldn't need to add any new ones here.